loadgen/README.md

```diff
@@ -103,3 +99,67 @@ with the reference models.
 
 For templates of how to do the above in detail, refer to code for the demos,
 tests, and reference models.
+
+
+## Submissions over the network
```

Comment:
@liorkhe Have we settled on an "official name" for this scenario? Is it "LoadGen Over Network" or "Inference over network" or something else?

Reply:
In the rules we used "Network Division" and LoadGen Systems over the Network so LoadGen Over Network is good for me.

Reply:
Title: LoadGen over Network